plugins/uploadwidget/plugin.js

```diff
@@ -252,6 +252,11 @@
 					oldStyle, newStyle;
 
 				loader.on( 'update', function( evt ) {
+					// Abort if the editor has been destroyed.
+					if ( widget.editor.status === 'destroyed' ) {
+						return;
```

Comment:
`return` does not prevent the continuation of file upload, it still happens in the background. If editor is destroyed, loader should be aborted (`loader.abort()`).